legend-engine-executionPlan-execution-store-relational-connection/src/main/java/org/finos/legend/engine/plan/execution/stores/relational/connection/driver/vendors/spanner/SpannerConnectionExtension.java

```diff
@@ -0,0 +1,15 @@
+package org.finos.legend.engine.plan.execution.stores.relational.connection.driver.vendors.spanner;
```

Comment:
This seems a bit off the convention.
Normally we register directly instead of extensions

if (managersByName == null)
                {
                    managersByName = ConcurrentHashMap.newMap();
                    register(new H2Manager());
                    register(new SnowflakeManager());
                    register(new BigQueryManager());
                    MutableList<ConnectionExtension> extensions = Iterate.addAllTo(ServiceLoader.load(ConnectionExtension.class), Lists.mutable.empty());
                    extensions.flatCollect(ConnectionExtension::getAdditionalDatabaseManager).forEach(DatabaseManager::register);
                }

Reply:
Hmmm .... I thought when we had discussed this we agreed to even remove BigQuery from here because we have the extension ?  

I like the idea of loading the managers via extension as it makes it easy to split db specific code into separate modules.

Reply:
Hey yes totally agree we would need extensions when we split into db specific
but a bit cautious as to till then we be in same state